split-the-stream/README.md

```diff
@@ -0,0 +1,89 @@
+# Ideas about splitting the source-git work and upstream work
+
+Some ideas, possibilities, pros and cons of moving out the source-git related work.
+
+## What does the source-git workflow mean?
+
+Must have:
+
+- If user creates a merge-request on the source-git repository:
+  - Create a matching merge-request to the dist-git repository.
+  - Sync the CI results from the dist-git merge-request to the source-git merge-request.
+- If the dist-git is updated, update the source-git repository.
+
+Should have:
+
+- If the source-git merge-request is updated, update the dist-git merge-request.
```

Comment:
Additionally: if the source-git MR is closed, close the dist-git MR—to make the mirroring complete.

Reply:
I'm not sure, whether there should be a mechanism when syncing the MRs to check that the dist-git MR was not modified since the last update, so that dist-git MRs edited by contributors are not overwritten.

Furthermore:
There should be means to trigger or re-do the update of dist-git MRs both through the service (with commands in comments) and through using the CLI.

Reply:
> through using the CLI.

Will this command change the merge request or the local dist-git repository?

Reply:
For the sake of simplicity it should update a local dist-git repo. It would fall on the developer to push those changes to update the MR.

Reply:
Updated.